LabManual.md

```diff
@@ -0,0 +1,13 @@
+# Lab Manual #
+
+## Setting up your Azure account ##
+
+You may activate an Azure free trial at https://azure.microsoft.com/en-us/free/.  
+
+If you have been given an Azure Pass to complete this lab, you may go to http://www.microsoftazurepass.com/ to activate it.  Please follow the instructions at https://www.microsoftazurepass.com/howto, which document the activation process.  A Microsoft account may have one free trial on Azure and one Azure Pass associated with it, so if you have already activated an Azure Pass on your Microsoft account, you will need to use the free trial or use another Microsoft account.  
```

Comment:
"Microsoft account may have one free trial on Azure and one Azure Pass associated with it" - Great adding that in, I've forgotten that on more than one occasion when troubleshooting.